vscode-integration/src/glsp-diagram-editor-provider.ts

```diff
@@ -59,6 +59,24 @@ export class GlspDiagramEditorProvider implements vscode.CustomEditorProvider<Gl
             command: GLSPCommand.LAYOUT,
             ...options
         });
+
+        GLSPCommand.registerActionCommand({
```

Comment:
The registration of the diagram-specific navigation commands should be done in `WorkflowGlspDiagramEditorContext`.
Maybe we should move the `registerCommands` method from the `EditorProvider` to the `GlspDiagramEditorContext`.